File: LLM-Translations/GPT-4/codenet/Python/C/s001486412.c
#include <stdio.h>

int main() {
    int n;
    scanf("%d", &n);
    double a[n];
    for (int i = 0; i < n; i++) {
        scanf("%lf", &a[i]);
    }
    double below = 0;
    for (int i = 0; i < n; i++) {
        below += 1 / a[i];
    }
    printf("%.9lf\n", 1 / below);
    return 0;
}
// 

File: LLM-Translations/GPT-4/codenet/Python/Go/s001486412.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var n int
	fmt.Scan(&n)
	a := make([]float64, n)
	for i := 0; i < n; i++ {
		fmt.Scan(&a[i])
	}
	below := 0.0
	for i := 0; i < n; i++ {
		below += 1 / a[i]
	}
	fmt.Println(1 / below)
}

// 
